Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml

```diff
@@ -633,17 +776,20 @@ inputs:
       - operator: uniq
   required: false
   description: The IP address to enrich.
+  playbookInputQuery:
 - key: InternalRange
   value:
     simple: inputs.InternalRange
   required: false
   description: 'A CSV list of IP address ranges (in CIDR notation). Use this list to check if an IP address is found within a set of IP address ranges. For example: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" (without quotes). If a list is not provided, will use default list provided in the IsIPInRanges script (the known IPv4 private address ranges).'
+  playbookInputQuery:
 - key: ResolveIP
   value:
     complex:
       root: inputs.ResolveIP
   required: true
   description: Whether to convert the IP address to a hostname using a DNS query (True/False).
+  playbookInputQuery:
 outputs:
 - contextPath: IP
```

Comment:
lets add dbotscore as well

Reply:
The row below includes the DBotScore of all indicators...
 contextPath: DBotScore
 description: Indicator, Score, Type, and Vendor.